src/components/tops/TopbarProductionList.vue

```diff
@@ -193,7 +193,7 @@ export default {
   border-bottom-left-radius: 5px;
   border-bottom-right-radius: 5px;
   margin-left: -6px;
-  max-height: 210px;
+  max-height: 50vh;
```

Comment:
This number should be a precise pixel number. Can you change it to 420px or similar?